packages/hardhat-core/test/internal/hardhat-network/provider/modules/debug.ts

```diff
@@ -159,7 +159,7 @@ describe("Debug module", function () {
     });
   });
 
-  describe("fork tests", function () {
+  describe("fork tests (pre-berlin)", function () {
```

Comment:
pre berlin or pre london?

Reply:
The block number is pre-Berlin. Maybe we also need some tests for a block that is post-Berlin but pre-London? I don't think it will add a lot of value though.